cli/types/cypress.d.ts

```diff
@@ -2838,6 +2838,30 @@ declare namespace Cypress {
     certs: PEMCert[] | PFXCert[]
   }
 
+  type RetryStrategyWithModeSpecs = RetryStrategy & {
```

Comment:
@AtofStryker yes, I agree it should be default `false`.

Reply:
@ryanpei I think what I am saying is setting it to `false` would break the API for experimental, but it essentially has the same effect as the default for the current API so it should be fine to leave it the way it is for experimental release

Reply:
@AtofStryker I see, so if they set retries `false`, they won't see any errors or anything, right?

Reply:
@ryanpei what I am saying is setting retries to `false` wouldn't be an option for experimental since it changes the API. To tweak retries, the user would need to provide `openMode` and `runMode` with booleans and provide an experimental strategy.

Reply:
I see so this is a breaking change for anyone who has set `retries: false`? There's no way to fallback to the old API, in this case?

Reply:
We can we would need to code for it though, which for an experimental release I am not sure of the value since it would be the same thing as the current default.